nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/field/ObjectLocalDateTimeFieldConverter.java

```diff
@@ -36,6 +37,11 @@
  */
 class ObjectLocalDateTimeFieldConverter implements FieldConverter<Object, LocalDateTime> {
     private static final long YEAR_TEN_THOUSAND = 253_402_300_800_000L;
+    private static final long YEAR_TEN_THOUSAND_SECONDS = 253_402_300_800L;
+    private static final char PERIOD = '.';
+
+    private static final long MICROSECOND_MULTIPLIER = 1_000;
+    private static final long MILLISECOND_MULTIPLIER = 1_000_000;
```

Comment:
Thoughts about using something like `MICROS_TO_NANOS` and `FRACTION_TO_MICROS_SECONDS` to be more self-explanatory?